.github/workflows/build_and_test.yml

```diff
@@ -344,13 +379,10 @@ jobs:
           echo "LD_LIBRARY_PATH=$LD_LIBRARY_PATH"
         fi
         ./install/bin/om_minverser --help
-        if [[ "${{matrix.python}}" == 'python'* ]]; then
+        if [[ "${{steps.env-vars.outputs.python_type}}" == 'cmake' ]]; then
           PYVER=$(python -c "import sys; print('.'.join(sys.version.split('.')[:2]))")
           if [[ "${{matrix.os}}" == 'windows'* ]]; then
-            export PYTHONPATH=$PWD/install/lib/site-packages
-            curl -LO https://github.com/lucasg/Dependencies/releases/download/v1.11.1/Dependencies_x64_Release.zip
-            unzip Dependencies_x64_Release.zip -d Dependencies
-            # TODO: This should not be necessary but let's see if it makes things green
+            # Ideally this would not be necessary but it makes things work
```

Comment:
```suggestion
            export PYTHONPATH=$PWD/install/lib/site-packages
            curl -LO https://github.com/lucasg/Dependencies/releases/download/v1.11.1/Dependencies_x64_Release.zip
            unzip Dependencies_x64_Release.zip -d Dependencies
            # Ideally this would not be necessary but it makes things work
```